tests/test_anon.py

```diff
@@ -90,9 +90,9 @@ def test_deleted_tags_are_removed(orig_anon_dataset):
             # TODO: Investigate why Date type are replaced instead of deleted
```

Comment:
```suggestion
            # Date type are replaced instead of deleted. See README.md.
```
and then delete the reference to issue #56

Reply:
Thanks !